File: Js/counter.js
function counter() {
    let count = 0;
    return {
        increment: () => {
            return count++;
        },
       decrement: () => {
            return count--;
        },
        getCount: () => count
    }
}


const c = counter();
console.log(c.getCount())
c.increment()
c.increment()
console.log(c.getCount())
c.decrement()
console.log(c.getCount())

File: Java/ThreadsPracticeApril16/Counter.java
package ThreadsPracticeApril16;

public class Counter {
    static int count = 0 ;
    static void incrementCount( ){
        count++;
    }
    public static void main(String []args) {
        Counter.incrementCount();
        Counter.incrementCount();
        System.out.println(count);
    }
}
